### PR TITLE
release: update manifest and helm charts for v0.0.18

### PR DESCRIPTION
**Pull request by aramase**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by aramase** [comment]
/test pull-secrets-store-csi-driver-e2e-windows

/hold
https://github.com/kubernetes/k8s.io/pull/1480 needs to be merged first

**Comment by tam7t** [comment]
I'm less familiar with the helm chart changes but the rest LGTM

File: Makefile [hunk 1]
```diff
@@ -248,5 +248,8 @@ promote-staging-manifest:
 	@cp -r manifest_staging/deploy .
 	@rm -rf charts/secrets-store-csi-driver
 	@cp -r manifest_staging/charts/secrets-store-csi-driver ./charts
-	@helm package ./charts/secrets-store-csi-driver -d ./charts/
-	@helm repo index ./charts --url https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts
+	@mkdir -p ./charts/tmp
+	@helm package ./charts/secrets-store-csi-driver -d ./charts/tmp/
+	@helm repo index ./charts/tmp --url https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts --merge ./charts/index.yaml
```
Comment (ritazh) [inline Makefile:253]: 👍  for updating this

**Pull request merged**